runtime/queries/hcl/highlights.scm

```diff
@@ -0,0 +1,84 @@
+; { key: val }
+;
+; highlight identifier keys as though they were block attributes
+; (object_elem (identifier) () (expression (variable_expr (identifier) @field)))
+
+; (object_elem val: (expression
+;   (variable_expr
+;     (identifier) @type.builtin (#any-of? @type.builtin "bool" "string" "number" "object" "tuple" "list" "map" "set" "any"))))
+
+(get_attr (identifier) @variable.builtin (#any-of? @variable.builtin  "root" "cwd" "module"))
+((identifier) @keyword (#any-of? @keyword "module" "root" "cwd" "resource" "variable" "data" "locals" "terraform" "provider" "output"))
+((identifier) @type.builtin (#any-of? @type.builtin "bool" "string" "number" "object" "tuple" "list" "map" "set" "any"))
+(variable_expr (identifier) @variable.builtin (#any-of? @variable.builtin "var" "local" "path"))
```

Comment:
helix doesn't have the `#any-of` predicate, so you'll need to convert these to an equivalent `#match?` in a format like

```tsq
(get_attr
  (identifier) @variable.builtin
  (#match? @variable.builtin "^(root|cwd|module)$"))
```

I think that should improve the highlights a bit :+1: